web/src/js/flow/utils.js

```diff
@@ -49,7 +49,14 @@ export var MessageUtils = {
         } else if (message === flow.response) {
             message = "response";
         }
-        return `/flows/${flow.id}/${message}/content` +  (view ? `/${view}` : '');
+        if (global.MITMWEB_STATIC) {
```

Comment:
I strongly agree - hence the regex to make `.json` optional, see https://www.debuggex.com/r/TVc9q4VcGlGO6znM. We can do `GET /flows.json`, but still do `POST /flows`. The static viewer only needs `GET`, so this covers our use case.